src/OrchardCore/OrchardCore.Infrastructure/Scripting/Files/FilesScriptEngine.cs

```diff
@@ -46,7 +46,7 @@ public object Evaluate(IScriptingScope scope, string script)
             }
 
             using var fileStream = fileInfo.CreateReadStream();
-            using var ms = new MemoryStream();
+            using var ms = MemoryStreamFactory.GetStream();
             fileStream.CopyTo(ms);
             return Convert.ToBase64String(ms.ToArray());
```

Comment:
ToArray() is doing an allocation. Use the GetSequence instead (GetBuffer() will also allocate)

Reply:
Where is `.GetSequance()`?